samples/peripheral/radio_test/README.rst

```diff
@@ -38,12 +38,12 @@ The sample also requires one of the following testing devices:
    You can perform the radio test also using a spectrum analyzer.
    This method of testing is not covered by this documentation.
 
-nRF21540 front-end module
-=========================
+Front-end module
+================
 
 .. include:: /includes/sample_dtm_radio_test_fem.txt
 
-You can configure the nRF21540 front-end module (FEM) transmitted power control, antenna output and activation delay using the main shell commands of the :ref:`radio_test_ui`.
+You can configure the front-end module (FEM) transmitted power control, antenna output, and activation delay using the main shell commands of the :ref:`radio_test_ui`.
```

Comment:
Please add a note to make it clear to the users, that each FEM has different interface and not all commands will work the same way, and some may not be supported, maybe something like:
`Note: each front-end module (FEM) has different capabilities and operating modes, so some commands may not be supported by a specific FEM and those supported may work differently on different FEMs.`

Reply:
Added